CHANGELOG.md

```diff
@@ -40,8 +40,12 @@ should now look like:
 evaluated at every flush interval, rather than once at startup. This makes it
 consistent with the behavior of `collection_jitter`.
 
+- postgresql plugins switched drivers to better handle oid and name data types.
```

Comment:
this is a bit of an implementation detail that doesn't need to be part of the changelog. What does this mean for the end-user of the postgres plugin?